tests/snuba/api/endpoints/test_organization_events_v2.py

```diff
@@ -4057,3 +4057,33 @@ def test_equation_field_limit(self):
         )
 
         assert response.status_code == 400
+
+    def test_count_if(self):
```

Comment:
Should we add a test case with a refining filter like "count_if(transaction.duration, less, 150): >2" or something?